scripts/raw_to_easymocap.py

```diff
@@ -0,0 +1,25 @@
+import os, sys, shutil
+
+DATA_TYPE = ['png','PNG','jpg','JPG']
```

Comment:
We should use lower case only, and for matching use something like `s.lower().endswith()`.

Reply:
Updated `DATA_TYPE` var to only use lowercase file extensions